Formula/alpine.rb

```diff
@@ -1,19 +1,14 @@
 class Alpine < Formula
   desc "News and email agent"
   homepage "https://repo.or.cz/alpine.git"
-  url "https://ftp.osuosl.org/pub/blfs/conglomeration/alpine/alpine-2.21.tar.xz"
-  mirror "https://fossies.org/linux/misc/alpine-2.21.tar.xz"
-  sha256 "6030b6881b8168546756ab3a5e43628d8d564539b0476578e287775573a77438"
-  revision 2
-
-  bottle do
```

Comment:
Do not remove or change the bottle block

Reply:
Sorry, I will fix this and avoid this mistake in the future.